data-science/README.md

```diff
@@ -1,30 +1,49 @@
- - Mandar a sua solução em código Python ou R. O uso de bibiliotecas é livre, mas você deve ser capaz de explicar o conceito dos algoritmos utilizados de maneira sucinta, ou seja, como eles funcionam e porque os escolheu para uso.
- - Além do desenvolvimento da solução é necessário mostrar métricas de avaliação da solução desenvolvida. É necessário apresentar a matriz de confusão do(s) modelo(s) desenvolvido(s), além de outras métricas de avaliação. Pontos extras para o uso do ROC/AUC. Além das métricas é necessário enviar a avaliação dos atributos do modelo e uma possível explicação para o porque o modelo apresentado é bom.
- - Enviar o modelo final para que seja possível testá-lo com um novo conjunto de dados. Para isso você deve garantir que o seu código seja reproduzível e explicar como deveremos rodá-lo no relatório.
+## O que você precisa fazer
+
+Utilizando um jupyter notebook, apresente a sua solução em python.
+
+1. Apresentar uma **Análise Exploratória dos Dados** (Exploratory Data Analysis / EDA) com gráficos.
+2. À partir da EDA acima, explicar decisões de pré processamento, limpeza, e remoção de dados. 
+3. Construir um modelo de classificação que retorne a probabilidade que um cliente tem de ser enviado para análise de crédito dado que ele foi pré-aprovado para o empréstimo com garantia de automóvel. O uso de bibiliotecas é livre, mas você deve ser capaz de explicar o conceito dos algoritmos utilizados de maneira sucinta, ou seja, como eles funcionam e porque os escolheu para uso.
+4. Além do desenvolvimento da solução é necessário mostrar métricas de avaliação da solução desenvolvida. Pense em métricas que fazem sentido para o problema em questão e na particularidade destes dados.
+5. Avaliação e interpretabilidade do modelo. Você deve responder quais variáveis são importantes para o modelo construido e porquê.
+6. Você deve garantir que o seu código seja reproduzível e explicar como deveremos rodá-lo em um arquivo `.md`.
 
-### Atenção
- - Os dados não estão normalizados. O cliente pode ter preenchido valores incomuns em alguns campos, como a renda.
- - Alguns dados estão faltando. Faz parte do exercício entender como que os dados faltantes serão tratados (ou justificar se não o serão).
- - Você poderá usar dados externos, desde que explique **detalhadamente** quais dados usou e como o fez.
- - Não há um limite fixo de tempo, porém é estimada que uma semana é mais que o suficiente para para resolver o problema.
 
-## Avaliação
+## Como entregar o desafio?
 Para nos enviar seu código, você pode:
- - Dar acesso ao seu repositório privado no [Gitlab](http://gitlab.com) para `creditaschallenge`.
+- Dar acesso ao seu repositório privado no [Gitlab](http://gitlab.com) para `creditaschallenge`.
 - Mandar um email para `ds-squad@creditas.com.br`
+
+
+### Dicas
+ - Os dados não estão normalizados. O cliente pode ter preenchido valores incomuns em alguns campos, como a renda.
```

Comment:
O ambiente de staging é fechado pra rede da Creditas, certo ? Se não for o caso não acharia interessante colocar uma dica de _Aconselhamos você entrar em <LINK_STAGING> e fazer uma simulação do nosso fluxo com dados de teste (não há necessidade de colocar seus dados reais) para saber um pouco mais sobre o nosso cadastro, o tipo de campo a ser preenchido, etc_